tests/gql_behave/tests/memgraph_V1/features/hops_limit.feature

```diff
@@ -0,0 +1,255 @@
+# Copyright 2024 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+Feature: Hops Limit
+
+    Scenario: Test hops limit DFS test01
+        Given graph "simple_binary_tree"
+        When executing query:
+            """
+            USING HOPS LIMIT 5 MATCH p=(a)-[:CONNECTED *]->(e) RETURN p
+            """
+        Then the result should be:
+            | p                                                                                                      |
+            | <(:Node {name: 'A'})-[:CONNECTED]->(:Node {name: 'B'})>                                                |
+            | <(:Node {name: 'A'})-[:CONNECTED]->(:Node {name: 'C'})>                                                |
+            | <(:Node {name: 'B'})-[:CONNECTED]->(:Node {name: 'D'})>                                                |
+            | <(:Node {name: 'A'})-[:CONNECTED]->(:Node {name: 'B'})-[:CONNECTED]->(:Node {name: 'D'})>              |
+            | <(:Node {name: 'B'})-[:CONNECTED]->(:Node {name: 'E'})>                                                |
+
+
+    Scenario: Test hops limit DFS test02
+        Given graph "simple_binary_tree"
```

Comment:
Are we sure that we want to run create nodes and edges on top of `simple_binary_tree` graph? Maybe it  was expected to be empty graph?

Reply:
That is a mistake, good catch.